pyproject.toml

```diff
@@ -62,7 +62,7 @@ select = [
 # Ignore `E402` and `F401` (unused imports) in all `__init__.py` files
 "__init__.py" = ["E402", "F401"]
 # ignore typing rules for tests
-"tests/*" = ["ANN201"]
+"tests/*" = ["ANN201", "PLR0913"]
```

Comment:
Thanks for checking, this ignores ruff linting rules [ANN201](https://docs.astral.sh/ruff/rules/missing-return-type-undocumented-public-function/) and [PLR0913](https://docs.astral.sh/ruff/rules/too-many-arguments/) for any files found within the test suite. During development I found that there were triggers for these checks and made a choice to avoid implementing a fix for them as it felt less useful to the work more generally.